### DISCUSSION TITLE
get_tweet() only returns id, text, not created_at in response.data object

### DISCUSSION THREAD

**Discussion by bybilal** [tweepy/tweepy#2086] [Questions] [answered]
Hello,

Surely I must be doing something wrong. 
The docs says **id**, **text**, and **created_at** should be in **response.data** but its only the id and the text
```
    client = tweepy.Client(access_token)

 # Retrieve the Tweet by its ID    
 tweet_id = "1631581739234754561" 
 tweet = client.get_tweet(id=tweet_id)
 print(tweet)
```

Can someone help? Thank you

response:

```
Response(data=<Tweet id=1631817370938535938 text='@tempst0 @0xgodking Reaching in 2 mins!'>, includes={}, errors=[], meta={})
```

**Comment by bybilal** [comment]
figured out its nested inside the obj

**Comment by Harmon758** [comment, marked as answer]
Besides the default `id` and `text` fields, you need to [use the `tweet_fields` parameter to specify the Tweet fields that you want returned](https://tweepy.readthedocs.io/en/v4.13.0/expansions_and_fields.html#tweet-fields).
